src/dist.rs

```diff
@@ -315,6 +345,79 @@ impl Dist {
     }
 }
 
+#[cfg(feature = "scss")]
+fn scss(
+    static_dir: &std::path::Path,
+    dist_dir: &std::path::Path,
+    options: sass_rs::Options
+) -> Result<()> {
+    use walkdir::{DirEntry, WalkDir};
+
+    fn is_scss(path: &std::path::Path) -> bool {
+        matches!(
+            path.extension().map(|x| x.to_str()).flatten(),
+            Some("sass") | Some("scss")
+        )
+    }
+
+    fn should_ignore(entry: &DirEntry) -> bool {
+        entry
+            .file_name()
+            .to_str()
+            .map(|x| x.starts_with("_"))
+            .unwrap_or(false) || entry.path().is_dir()
+    }
+
+    log::trace!("Generating dist artifacts");
+    let walker = WalkDir::new(&static_dir).into_iter();
+    for entry in walker
+        .filter_map(|x| match x {
+            Ok(x) => {
+                if !should_ignore(&x) {
+                    log::debug!("Will be handled: `{}`", &x.path().display());
+                    Some(x)
+                } else {
+                    log::debug!("Will be ignored: `{}`", &x.path().display());
+                    None
+                }
+            },
+            Err(_err) => {
+                log::error!("could not walk into directory: `{}`", &static_dir.display());
+                None
+            }
+        })
+    {
+        let file_path = entry.path();
+        let dist_path = dist_dir.join(file_path.strip_prefix(&static_dir).unwrap());
+
+        if is_scss(file_path) {
+            let css_path = dist_path
+                .with_extension("css");
+
+            match sass_rs::compile_file(file_path, options.clone()) {
+                Ok(css) => {
+                    let _ = fs::create_dir_all(css_path.parent().unwrap());
```

Comment:
This should probably be moved outside the `if` block because you always create the directories to it in both scenarios

Reply:
Even if `sass_rs::compile_file()` return an error?

Reply:
who cares